Resources/Prototypes/_SpaceCats/Entities/Structures/Wallmount/Signs/posters.yml

```
@@ -0,0 +1,877 @@
+  components:
+  - type: Sprite
+    state: poster_whatdv_legit #Спрайт от Празата
+
+- type: entity
+  parent: PosterBases
+  id: PosterUnclePrazLegit
+  name: We need you!
+  suffix: DV
+  components:
+  - type: Sprite
+    state: poster_unclepraz_legit #Спрайт от Празата
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)